src/components/Book.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = (props) => {
+  const {
+    title, category,
+  } = props;
+
+  const BooksData = {
+    // gender: 'Action',
+    // title: 'The Hunger Games',
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.